tests/integration/api/test_event_stream.py

```diff
@@ -463,3 +467,41 @@ def test_list_event_stream_instances(
     assert len(data) == len(instances)
     assert data[0]["name"] == instances[0].name
     assert data[1]["name"] == instances[1].name
+    assert data[0]["queue_name"] == "activation"
+    assert data[1]["queue_name"] is None
+
+
+@pytest.mark.django_db
+def test_retrieve_event_stream_instance(
+    client: APIClient,
+    default_de: models.DecisionEnvironment,
+    default_user: models.User,
+):
+    args = {"limit": 5, "delay": 1}
+    event_stream = models.EventStream.objects.create(
+        name="test-event_stream-1",
+        source_type="ansible.eda.range",
+        source_args=args,
+        user=default_user,
+        decision_environment_id=default_de.id,
+    )
+
+    instance = models.RulebookProcess.objects.create(
```

Comment:
Should these be exported to fixtures? I see it being used in multiple test functions.

Reply:
Honestly, I've not look at it. I was directed to the test. Probably should be. Since we are not even executing these tests, probably it is worth to refactor them later. do you think?

Reply:
@Alex-Izquierdo I'm fine with refactoring them later as part of its own story/epic; this is not a merge blocker.